### ISSUE TITLE
Main Menu music doesn't loop

### ISSUE THREAD

**Issue by Richard-L** [LostArtefacts/TRX#1652]
Currently once the track finishes, it does not start again.

I guess this is partly due to the missing demo modes https://github.com/LostArtefacts/TR2X/issues/7 where upon main menu return it restarts the music.

**Comment by aredfan** [comment]
In my case, the demo mode does start when the Main Menu is idle for 30 seconds.

When the demo mode finishes and I get returned to the Main Menu, there's no music.  I've confirmed that the bug is caused by LostArtefacts/TRX#1614.

**Edit:** You're right - the Main Menu music doesn't loop.  I went onto the Main Menu and kept pressing left/right to prevent the demo mode from starting until the music track ended, to be sure.
